modules/proxy-certificates.adoc

```diff
@@ -0,0 +1,114 @@
+
+// Module included in the following assemblies:
+//
+// * authentication/understanding-certificates.adoc
+
+[id="proxy-certificates_{context}"]
+= Proxy certificates
+
+== Purpose
+
+The `trustedCA` field of `proxy` is a reference to a ConfigMap that contains a
+user-provided trusted certificate authority (CA) bundle. This bundle is merged
+with the Red Hat Enterprise Linux CoreOS (RHCOS) trust bundle and injected into
+the trust store of platform components that make egress HTTPS calls. For
+example, `image-registry-operator` calls an external image registry to download
+images. If `trustedCA` is not specified, only the RHCOS trust bundle is used for
+proxied HTTPS connections. Provide custom CA certs to the RHCOS trust bundle if
+you want to use your own certificate infrastructure.
+
+The `trustedCA` field should only be consumed by a proxy validator. The
+validator is responsible for reading the certificate bundle from required key
+`ca-bundle.crt` and copying it to a ConfigMap named `trusted-ca-bundle` in the
+`openshift-config-managed` namespace. The namespace for the ConfigMap referenced
+by `trustedCA` is `openshift-config`:
+
+[source,yaml]
+----
+apiVersion: v1
+	kind: ConfigMap
+ metadata:
+	 name: user-ca-bundle
+	 namespace: openshift-config
+	 data:
+	   ca-bundle.crt: |
+	     -----BEGIN CERTIFICATE-----
+	     Custom CA certificate bundle.
+	     -----END CERTIFICATE-----
+----
+
+== File path
+
+The user-provided trust bundle is represented as a ConfigMap. The ConfigMap is
+mounted into the filesystem of platform components that make egress HTTPS calls.
+Typically, Operators mount the ConfigMap to
+`/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem`, but this not required by
+proxy and is at the Operator's discretion.
+
+If using the RHCOS trust bundle, place CA certificates in
+`/etc/pki/ca-trust/source/anchors`.
+
+== Expiration
+
+The user sets the expiration term of the user-provided trust bundle.
+
+The default expiration term is defined by the CA certificate itself. It is up to
+the CA administrator to configure this for the certificate before it can be used
+by {product-title} or RHCOS.
```

Comment:
You might add a note; that we don't monitor for when CA's expire; however due to the long life of CA's this is generally not a problem, however; users may need to update the trust-bundle from time to time, and we should probably have a documented process to do that. 

Linking back to https://github.com/openshift/openshift-docs/pull/18254/files#diff-870826a1e88c9daf50d96b53b3fe3592R11 may suffice for that.